assets/js/components/HostDetails/HostInfoBox.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import ListView from '@components/ListView';
+import ProviderLabel from '@components/ProviderLabel';
+
+function HostInfoBox({ provider, agentVersion }) {
+  return (
+    <div className="my-4 bg-white shadow rounded-lg px-8 py-4">
+      <ListView
+        orientation="vertical"
+        data={[
+          {
+            title: 'Provider',
+            content: provider,
+            render: (content) => <ProviderLabel provider={content} />,
+          },
+          { title: 'Agent version', content: agentVersion },
+          {
+            title: '',
```

Comment:
Hey any reason to keep empty title and content besides css ? 
Removing it seems not to harm :D

Reply:
little hack to have it as Jurgen designed it :see_no_evil:

Reply:
@jagabomb Could you have a look into this design?
It doesn't follow the same spacing than the Cluster check selection, where we as well have 2 elements.
I personally wouldn't add this space, and be consistent in what we have in the cluster details view. We could even consider putting the provider in 2nd position

Reply:
Could you at least put a comment saying that this is a temporary hack?